print_dynsymtab.cc

```diff
@@ -0,0 +1,14 @@
+#include "elf_binary.h"
```

Comment:
How about to mention this tool works even without shdrs unlike `nm -D --with-symbol-versions`?

Reply:
It sounds good idea. I wrote a description of `print_dynsymtab` at the head of `print_dynsymtab.cc`.